requirements.txt

```diff
@@ -5,7 +5,7 @@ futures
 # https://github.com/tensorflow/tensorflow/issues/44467
 h5py~=2.10.0
 Keras==2.1.5
-ivadomed==2.7.2
```

Comment:
Actually, quick question: Why do we pin `ivadomed` at all? Is it possible we want to leave it unpinned, as to always use the latest version of ivadomed during development?

Reply:
>  Is it possible we want to leave it unpinned, as to always use the latest version of ivadomed during development of SCT?

great point-- i'm happy to leave it at the latest version-- can you do the change pls? thanks